packages/seal/src/shamir.ts

```diff
@@ -241,9 +241,11 @@ export class Polynomial {
 
 	/** Evaluate the polynomial at x. */
 	evaluate(x: GF256): GF256 {
-		return this.coefficients
-			.toReversed()
-			.reduce((sum, coefficient) => sum.mul(x).add(coefficient), GF256.zero());
+		let result = GF256.zero();
```

Comment:
unless there is a bug, i'd prefer to revert it (already audited)

Reply:
ok, not a bug, but an efficiency improvement (2x)